src/GeositeFramework/plugins/layer_selector_v2/State.js

```diff
@@ -344,6 +375,34 @@ define([
 
             serialize: function() {
                 return this.savedState;
+            },
+
+            setLayerOpacity: function(layerId, opacity) {
+                var layerItem = _.find(this.savedState.layerOpacity, function(layer) {
+                    return layer.layerId === layerId;
+                });
```

Comment:
Consider replacing with `_.findWhere` to match how it's done in `getLayerOpacity`.